inst/tutorials/non_normal_models.Rmd

```diff
@@ -9,22 +9,58 @@ knitr::opts_chunk$set(echo = TRUE)
 library(learnr)
 ```
 
-## A Simple Linear Model with uniformly distributed errors.
+## A Simple Linear Model with Normally distributed errors
 
-In this tutorial, we will...
+We start our discussion by considering the following model, which you should be familiar with by now:
 
-----
+$$Y = 5 + 2X + \varepsilon, \quad \varepsilon \sim \mathcal{N}(0, 2^2)$$
 
-We start our discussion by considering the following model:
+We will write a function called `simulate_1_sample_normal` that draws a sample of a given size from the population and returns it as a `data.frame`.
+
+```{r simulate_1_sample_normal}
+simulate_1_sample_normal <- function(size){
+  x = rnorm(size, 3, 3)
```

Comment:
it's good to write code.
but each line should be commented clearly explaining what it does